api/info/service.go

```diff
@@ -319,6 +324,59 @@ func (i *Info) Uptime(_ *http.Request, args *UptimeRequest, reply *UptimeRespons
 	return nil
 }
 
+type ACP struct {
+	SupportWeight json.Uint64         `json:"supportWeight"`
+	Supporters    set.Set[ids.NodeID] `json:"supporters"`
+	ObjectWeight  json.Uint64         `json:"objectWeight"`
+	Objectors     set.Set[ids.NodeID] `json:"objectors"`
+	AbstainWeight json.Uint64         `json:"abstainWeight"`
```

Comment:
Can we make this a single `map[ids.NodeID]Vote` instead? `Vote` can contain stake weight, and abstain/support/reject